src/Simplex/Messaging/Server/CLI.hs

```diff
@@ -174,6 +174,24 @@ iniOnOff section name ini = case lookupValue section name ini of
   Right s -> error . T.unpack $ "invalid INI setting " <> name <> ": " <> s
   _ -> Nothing
 
+withPrompt :: String -> IO a -> IO a
+withPrompt s a = putStr s >> hFlush stdout >> a
+
+onOffPrompt :: String -> Bool -> IO Bool
+onOffPrompt prompt def =
+  withPrompt (prompt <> if def then " (Yn): " else " (yN): ") $
```

Comment:
```suggestion
  withPrompt (prompt <> if def then " (Y/n): " else " (y/N): ") $
```

Reply:
it's not exactly the same as above - although might be confusing. Here, there is the default, in the above case the input is required.